content/docs/command-reference/import.md

```diff
@@ -114,11 +111,9 @@ Importing 'data/data.xml (git@github.com:iterative/example-get-started)'
 ```
 
 In contrast with `dvc get`, this command doesn't just download the data file,
-but it also creates an import stage
-([`.dvc` file](/doc/user-guide/dvc-files-and-directories#dvc-files)) with a link
-to the data source (as explained in the description above). (This import stage
-can later be used to [update](/doc/command-reference/update) the import.) Check
-`data.xml.dvc`:
+but it also creates an import stage (`.dvc` file) with a link to the data source
```

Comment:
Note for self: replace all these "import stage" term instances in a separate PR.